### DISCUSSION TITLE
GitHub Sponsors Application Status and Payoneer Bank Account Usage Question

### DISCUSSION THREAD

**Discussion by edevsh** [community/community#140368] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I submitted a request to join the GitHub Sponsors program more than a few days ago, possibly even over two weeks, but it is still pending approval. I understand that the review process can take some time; however, I wanted to follow up to see if anyone else has experienced similar delays or could provide insight into the typical timeline for approval. **Is there anything I can do from my side to expedite this process?**

Additionally, I have a related question about setting up a payment method. I plan to add a Payoneer bank account **(specifically a USD account)** to my GitHub Sponsors profile. However, **Payoneer has indicated that this account only supports local payments**. **Given that GitHub Sponsors payments may involve international transactions, could using this Payoneer USD account potentially cause any issues?** I want to ensure that there are no restrictions or complications when it comes to receiving payments.

I appreciate any advice or experiences you can share regarding both the application process and setting up the Payoneer account.

Thank you for your time and support!

![image](https://github.com/user-attachments/assets/a2756e7e-f235-4d0e-9b7c-cc6d16d32422)

**Comment by CodeNKoffee** [comment]
I was hoping to get an answer here :(

**Comment by fyodorio** [comment]
@shahmal1yev have you acquired any experience of that eventually, to share here probably? Also interested in the question.

**Comment by edevsh** [comment, marked as answer]
Hello everyone 👋

Sorry for the late response — I couldn't find the support ticket I wanted to share in my GitHub support archive, and ended up searching for a while in the trash folder.

Anyway, I wanted to walk you through the full process I experienced in as much detail as possible, since there’s very little information on this topic and it might be useful to others.

Since I couldn’t get a clear answer within this discussion thread, I contacted GitHub Support directly. We had a long but productive exchange, and I’ll now share the key questions and the answers I received — along with screenshots.

---

### 1. How long does GitHub Sponsors approval take, and why can it be delayed?

As you may know, the **"Get Sponsored"** page offers two ways to initiate the process:

- Bank Account  
- Fiscal Host

At first, I tried applying via several Fiscal Hosts by creating accounts with them so I could use those details in my GitHub Sponsors application. However, that didn’t work out for a few reasons:

- Some rejected me because I didn’t have a popular repo or a notable open-source project,
- Some said I wasn’t affiliated with a well-established organization,
- And some declined the application for other various reasons.

So, I decided to proceed with the **Bank Account** method instead.

> [!IMPORTANT]  
> If you're applying for the first time and your country is not supported by Stripe for individual accounts,  it's strongly recommended **not to submit** the form.  Otherwise, your request might silently fall into a **limbo state**, and you won’t receive any updates.  
> 
> (Perhaps GitHub monitors Stripe updates for those countries and might eventually reach out — but don’t rely on it.)

If you're in a supported country, you’ll receive a confirmation email as soon as your application is submitted. You can then access the now-deprecated Sponsors Dashboard to complete your profile and send it for review. Usually, they respond quite quickly.

In my case, even though I reached out to support (which helped speed things up), my application had been silently stuck in a **suspended/limbo** list. 

I had even changed my country multiple times to a supported one, but received no reply. Eventually, a support agent fixed this **manually**.  
Here’s a screenshot confirming the limbo state: 

![Screenshot From 2025-04-03 16-19-26](https://github.com/user-attachments/assets/b3adc588-bacf-4f4a-8369-9979fad2871f)

---

### 2. Payoneer USD account only supports local transfers — is it compatible with GitHub Sponsors?

Let me share some insights from my communication with support. Initially, they mistakenly assumed I was applying on behalf of an organization rather than a personal account, so I had to clarify that before continuing.

As you’ll see in the two screenshots I’ve included, all that really matters is that your GitHub Sponsors setup is tied to a verified **Stripe Connect account**. That’s enough to proceed.

Now here’s the important part:  
Since Payoneer’s USD account only supports local (ACH) payments, I figured that using my **EUR Payoneer account** might be a better fit for the country I selected during the application — and it worked out exactly as I had hoped.  
(The EUR account is one of Payoneer’s free “receiving accounts” and doesn’t require any setup fees.)

Everything has been smooth so far. And if I ever run into any payout-related issues, I’m confident either Stripe or Payoneer will assist — although I honestly don’t expect any trouble.

Screenshots: 

![Screenshot From 2025-04-03 16-24-07](https://github.com/user-attachments/assets/410f186e-2dc5-4f63-90e5-a5725456e2d6)

![Screenshot From 2025-04-03 16-26-11](https://github.com/user-attachments/assets/d465c9e9-bdaf-4f2b-aac0-9baa2fe411ed)

![image](https://github.com/user-attachments/assets/1d979f30-727a-4a36-9cb8-57a6b623abc8)

---

I hope this was helpful to anyone going through the same process.  
Feel free to reach out if you have any questions — happy to share more!

---

@CodeNKoffee
@fyodorio

**Reply by CodeNKoffee** [reply]
First of all, I'd like to express my gratitude and how grateful I am for you taking the sharing the responses and digital journey you embarked on with the GitHub Support Team!

Second of all, so using a Payoneer account is ok but of course Stripe won't accept that account if I'm not a resident in the US or EUROPE according to the bank account it got, right?

Also, the supported countries list for Stripe Connects is different from the main Stripe account; that's what I inferred from Rowan (GitHub Support Team) saying when he said Connects is different from the Main account. The thing is, I've already tied in the past my Egyptian bank account to Stripe somehow and it works but I hadn't setup a main Stripe Account since Egypt is not listed; to my surprise though, on Connects it accepted Egyptian Bank Account tied on Stripe and used it on Connects directly.

**Reply by edevsh** [reply]
I'm happy to hear this was helpful!

Just to share something personal — I'm actually not residing in the country currently associated with either my Stripe or Stripe Connect accounts.

Despite this, by leveraging the account details provided by Payoneer and following up with Stripe Support over an extended email exchange, I was able to get my GitHub Sponsors setup fully working.

Everything seems to be functioning properly now. The only minor drawback is having to deal with taxes from multiple services

**Reply by fyodorio** [reply]
Thanks a lot for sharing the detailed story, that will definitely help 👍

**Comment by shahmeer78** [comment]
Payoneer withdrawal services are now available, with no minimum limit required. You can even withdraw from unverified accounts and blocked balances
WhatsApp Contact #
923087180089
